carma-messenger-core/docker/install_dependencies.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+#  Copyright (C) 2025 LEIDOS.
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+
+set -exo pipefail
+
+dir=~
+BRANCH=develop  # The script will use this unless the -b flag updates it
+while [[ $# -gt 0 ]]; do
+      arg="$1"
+      case $arg in
+            -b|--branch)
+                  BRANCH=$2
+                  shift
+                  shift
+            ;;
+            -r|--root)
```

Comment:
Remove root argument since it is not used

Reply:
Using the root argument is being used to find the path to the install_dependencies script here